config.yml

```diff
@@ -135,3 +135,28 @@ languages:
         - url: 'https://github.com/martignoni/moodle-tool-guide'
           name: github
           weight: 2
+
+  eu:
+    languageName: Euskara
+    title: Moodleko Tresnen Gida
+    slogan: Moodleko Tresnen Gida irakasle eta hezitzaileentzat
+    description: 'Moodleko Tresnen Gida: 3 edo bertsio berriago bat erabiltzeko irakasle, hezitzaile eta trebatzaileentzat diseinatutako erreferentziazko taula da.'
+    contentDir: content/eu
+    date_format: 2006 January 2
+    copyright: '[Nicolas Martignoni](https://blog.martignoni.net/a-propos/)-ek egina – Webgunea __[moodletoolguide.net](https://moodletoolguide.net)__ [Creative Commons BY-NC-SA 4.0 International](https://creativecommons.org/licenses/by-nc-sa/4.0/) lizentziapean argitaratu da.
+
+
+      Tresna hau [Eastern Institute of Technology](https://www.eit.ac.nz/)-ko (Zeelanda Berria) [Joyce Seitzinger](https://twitter.com/catspyjamasnz)-en ideia izan zen, 2020ko maiatzean Moodle 3.x-erako [Nicolas Martignoni][nm]-eko egokitua [Gavin Henrick](https://twitter.com/ghenrick)-ek 2 bertsiorako egindako lanean oinarrituta.'
```

Comment:
Please only translate this This line should be a translation of "An idea by Joyce Seitzinger of the Eastern Institute of Technology (NZ), updated in May 2020 for Moodle 3.x by Nicolas Martignoni and translated into Basque by Iñigo Zendegi.", obviously with the links.

See below too.

Reply:
I've updated that text both here and in content/eu/about/index.md